osu.Game/Users/Drawables/UpdateableAvatar.cs

```diff
@@ -69,14 +69,20 @@ protected override Drawable CreateDrawable(User user)
             if (user == null && !showGuestOnNull)
                 return null;
 
-            var avatar = new ClickableAvatar(user)
+            if (!openOnClick)
+            {
+                return new DrawableAvatar(user)
+                {
+                    RelativeSizeAxes = Axes.Both,
+                };
+            }
+
+            return new ClickableAvatar(user)
             {
                 OpenOnClick = openOnClick,
```

Comment:
can't be reached. also this is the only use, so needs to be removed if this is the path forward.